doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -183,88 +183,89 @@ Values are provided in kilobytes (KB).
       +----------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Template <matter_template_sample>` (Debug + LTO)         |            48 |               800 |              4 |         32 |         884 |                             301 |
+      | :ref:`Template <matter_template_sample>` (Debug)               |            48 |               885 |              4 |         32 |         969 |                             296 |
       +----------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Template <matter_template_sample>` (Release)             |            48 |               788 |              4 |         32 |         872 |                             289 |
+      | :ref:`Template <matter_template_sample>` (Debug + LTO)         |            48 |               808 |              4 |         32 |         892 |                             300 |
       +----------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Thermostat <matter_thermostat_sample>` (Debug)           |            48 |               923 |              4 |         32 |        1007 |                             298 |
+      | :ref:`Template <matter_template_sample>` (Release)             |            48 |               796 |              4 |         32 |         880 |                             290 |
       +----------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Thermostat <matter_thermostat_sample>` (Debug + LTO)     |            48 |               840 |              4 |         32 |         924 |                             302 |
+      | :ref:`Thermostat <matter_thermostat_sample>` (Debug)           |            48 |               852 |              4 |         32 |         936 |                             301 |
       +----------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Thermostat <matter_thermostat_sample>` (Release)         |            48 |               821 |              4 |         32 |         905 |                             290 |
+      | :ref:`Thermostat <matter_thermostat_sample>` (Debug + LTO)     |            48 |               852 |              4 |         32 |         936 |                             301 |
```

Comment:
Let's remove this line or one above.

Reply:
for lock and thermostat samples LTO is enabled by default in debug configuration. removed debug for both of these